backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -290,6 +290,42 @@ class TestOperator extends WholeStageTransformerSuite {
     assert(df.queryExecution.executedPlan.find(_.isInstanceOf[ProjectExecTransformer]).isDefined)
   }
 
+  test("Test round function") {
+    val df = spark.sql("SELECT round(cast(l_orderkey as int), 2) from lineitem limit 1")
+    val result = df.collect()
+    assert(result.length == 1)
+    val expected = Seq(Row(1))
+    TestUtils.compareAnswers(result, expected)
```

Comment:
Maybe do a rebase after @lviiii's PR being merged to use the new API to compare answer and verify the query plan.

Reply:
Done